### PR TITLE
Change build target to ES5

### PR DESCRIPTION
**Pull request by pheew**
This changes the build target to ES5.

Currently the lib will break when used in IE11 because `class` isn't transpiled because of the `ES6` target.

### CODE REVIEW

File: src/validation_error.ts [hunk 1]
```diff
@@ -7,13 +7,14 @@ interface ValidationErrorJsonReport {
 }
 
 
-export class ValidationError extends Error {
+export class ValidationError {
     private readonly key: ValidationErrorKey | undefined;
     private readonly nestedErrors: ValidationError[];
+    private readonly message: string;
```
Comment (tbknl) [inline src/validation_error.ts]: Could you make `message` public?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-	"name": "specified",
-	"version": "0.1.0",
+	"name": "@grandvision/specified",
+	"version": "0.2.1",
```
Comment (tbknl) [inline package.json]: Version bump happens in separate commit.

File: .gitignore [hunk 3]
```diff
@@ -1,2 +1,3 @@
 node_modules/
 dist/
+.idea
```
Comment (tbknl) [inline .gitignore]: Please no IDE specific ignores in the repo. Use global gitigniore for that.

**Comment by pheew** [comment]
@tbknl I've updated the PR, could you please re-check?

**Pull request merged**